### PR TITLE
Extract reporting classes from core into new rendering subproject in core-runtime

### PR DESCRIPTION
**Pull request by tresat**

### CODE REVIEW

**Comment by tresat** [comment]
@bot-gradle test this

File: platforms/core-runtime/rendering/build.gradle.kts
Comment (big-guy) [inline platforms/core-runtime/rendering/build.gradle.kts:1]: Maybe instead of rendering, this could be `report-rendering`?

Comment (tresat) [reply platforms/core-runtime/rendering/build.gradle.kts:1]: Sure.  I'll update it.

**Comment by tresat** [comment]
@bot-gradle test this

**Comment by tresat** [comment]
@bot-gradle test this

**Review by big-guy — APPROVED** [review]

**Comment by tresat** [comment]
@bot-gradle test and merge

**Pull request merged**